src/parser/jobs/sch/Aetherflow.js

```diff
@@ -0,0 +1,86 @@
+import React, {Fragment} from 'react'
+import {Table} from 'semantic-ui-react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+// Flow needs to be burnt before first use - 15s is optimal for first cast
+const FIRST_FLOW_TIMESTAMP = 10000
+
+export default class Aetherflow extends Module {
+	static handle = 'aetherflow'
+	static dependencies = [
+		'checklist',
+		'cooldowns',
+		'enemies',
+		'suggestions',
+	]
+
+	constructor(...args) {
+		super(...args)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onComplete() {
+		// Checklist rule for aetherflow cooldown
+		this.checklist.add(new Rule({
+			name: <Fragment>Use all <ActionLink {...ACTIONS.AETHERFLOW} /> stacks within a 45 second window.</Fragment>,
+			description: `
+				The level 68 trait, Quickened Aetherflow, reduces your Aetherflow cooldown by 5 seconds after using a single stack of aetherflow.
+				Using all your stacks before the cooldown is up effectively reduces the cooldown from 60 to 45 seconds,
+				which helps with mana management.
+			`,
+			requirements: [
+				new Requirement({
+					name: <Fragment><ActionLink {...ACTIONS.AETHERFLOW} /> cooldown uptime</Fragment>,
+					percent: (this.getQuickenedAetherflowOnCooldown() / (this.parser.fightDuration - FIRST_FLOW_TIMESTAMP)) * 100,
+				}),
+			],
+		}))
+	}
+
+	getQuickenedAetherflowOnCooldown() {
```

Comment:
Should keep it using `this.cooldowns.getTimeOnCooldown`.

You can call `this.cooldowns.reduceCooldown` when players cast CDR skills, it'll automatically calculate the correct time on cooldown once that's implemented.